tests/testthat/test-population.R

```diff
@@ -44,6 +44,26 @@ test_that("It can add user defined variability using an existing parameter", {
   expect_identical(population$getParameterValues(venousBloodVolume), values)
 })
 
+test_that("It can add user defined variability using values with NaN", {
+  population <- loadPopulation(populationFileName)
+  parameterPath <- "Organism|MyParameterWithNaN"
+  values_with_NAN <- c(1:10) * 2.5
+  values_with_NAN[2] <- NaN
+  population$setParameterValues(parameterPath, values_with_NAN)
+  expect_true(population$has(parameterPath))
+  expect_identical(population$getParameterValues(parameterPath), values_with_NAN)
+})
+
+test_that("It can add user defined variability using values with NA", {
+  population <- loadPopulation(populationFileName)
+  parameterPath <- "Organism|MyParameterWithNA"
+  values_with_NA <- c(1:10) * 2.5
+  values_with_NA[2] <- NA
+  population$setParameterValues(parameterPath, values_with_NA)
+  expect_true(population$has(parameterPath))
+  expect_identical(population$getParameterValues(parameterPath), values_with_NA)

```

Comment:
@Yuri05 This is quite suprising to me. The getParameterValues returns NA even though the numbers are stored as NaN. Something happens in rCLR that I do not understand

Reply:
could be a wrong conversion in rClr...

Reply:
how would it know that one was NA and the other one was NaN. I checked in .NET with console print and we only have NaN
At any rate it works so :)